joint_state_publisher:
  ros__parameters:
    source_list: ["/joint_states"]  # Eklem durumlarını alacağı topic
    publish_frequency: 50.0  # Yayınlama frekansı (Hz)
